packages/api/src/index.ts

```diff
@@ -1,31 +1,24 @@
-import { derive as ormlDerives } from '@open-web3/orml-api-derive';
+import type { ApiOptions } from '@polkadot/api/types';
+import type { RegistryTypes } from '@polkadot/types/types';
+
 import { derive as acalaDerives } from '@acala-network/api-derive';
-import {
-  rpc as acalaRpc,
-  types as acalaTypes,
-  typesAlias as acalaTypesAlias,
-  typesBundle as acalaTypesBundle,
-  signedExtensions as acalaSignedExtensions,
-  lookupTypes as acalaLookupTypes
-} from '@acala-network/types';
-import { ApiOptions } from '@polkadot/api/types';
-import { RegistryTypes } from '@polkadot/types/types';
+import { lookupTypes as acalaLookupTypes, rpc as acalaRpc, signedExtensions as acalaSignedExtensions, types as acalaTypes, typesAlias as acalaTypesAlias, typesBundle as acalaTypesBundle } from '@acala-network/types';
+import { derive as ormlDerives } from '@open-web3/orml-api-derive';
```

Comment:
are we able to fetch these types from chain?

Reply:
there're some custom types we still need to define. can't say which one isn't needed. need to go through all one by one